pilot/docker/Dockerfile.proxyv2

```diff
@@ -18,6 +18,7 @@ RUN chmod 2755 /usr/local/bin/envoy /usr/local/bin/pilot-agent
 ADD envoy_pilot.yaml.tmpl /etc/istio/proxy/envoy_pilot.yaml.tmpl
 ADD envoy_policy.yaml.tmpl /etc/istio/proxy/envoy_policy.yaml.tmpl
 ADD envoy_telemetry.yaml.tmpl /etc/istio/proxy/envoy_telemetry.yaml.tmpl
+ADD nodeagent-sds-cert.pem /etc/istio/nodeagent-sds-cert.pem
```

Comment:
I think we need the private key as well, right?

Reply:
Client(envoy) only needs cert. node agent needs key.

Reply:
oh, i thought it's for node agent (because it says nodeagent cert), never mind.